Browser-Storage.md

```diff
@@ -0,0 +1,90 @@
+## Store data in your web browser's storage
+
+In order to store data coming from and created by your web application, you do not necessarily need a database.
+The respective features are supported by Chrome (version 4 and higher), Mozilla Firefox (version 3.5 and higher) and Internet Explorer (version 8 and higher), and a range of other browsers including those of iOs and Android.
+
+There are two main possibilities for browser storage:
+
+### 1. localStorage
+
+Any content/data saved to the localStorage object will be available after the browser has been restarted (closed and opened again).
```

Comment:
Format the keywords such as "localStorage" as code using back ticks. For ex. `localStorage` 
Do this else where